### ISSUE TITLE
Switch to SafariViewController

### ISSUE THREAD

**Issue by jrk** [duckduckgo/ios-search-and-stories#110]
I'm interested in this as an alternative to Google.app on iOS 9. I suspect I'm not the only one who'd love to see content blocker and cookie integration by switching to a SafariViewController.

**Comment by sreilly** [comment]
:+1:

**Comment by tagawa** [comment]
Also a request on Twitter for adding support for Safari's content blocker: https://twitter.com/moisterrific/status/676594548320043008

**Comment by erikvdwal** [comment]
Note that `SFSafariViewController` does not allow adding custom UI elements like a favorite button, which means you'd lose the ability to favorite a page inside the app. One way around this may be to add a custom `UIActivity` that can save a page as a favorite.

**Comment by jrk** [comment]
Indeed. And related, my assumption was that the search results would probably still be in the existing WebKit controller, but all of the result links would be redirected to an SFSafariViewController. (This doesn't address favoriting of pages while browsing, but can address any desire for custom behavior up through the search results pages.)

**Comment by AngeloStavrow** [comment]
I considered the idea of a UIActivity too, but it does make favouriting an article a less discoverable.

Since a Safari view controller can be initialized with Reader mode active, does it make sense to add an option for this in the settings screen (or combine it with enabling Readability)? This might mean having to "de-brand" the feature—call it Easy-Reading Mode or Lightweight Mode or whatever.
